spec/requests/sessions_spec.rb

```diff
@@ -1,27 +1,23 @@
 require "rails_helper"
 
 RSpec.describe "Sessions", type: :request do
-  around do |example|
-    host! ENV["PLACEMENTS_HOST"]
-    example.run
-    host! nil
-  end
-
   describe "POST /auth/developer/callback" do
     it "returns http success" do
-      placements_user = create(:placements_user)
-      post auth_developer_callback_path,
+      claims_user = create(:claims_user)
```

Comment:
Any reason why this was changed from `placements_user` to `claims_user`? Was this a meaningful change?

Reply:
Changing the default service to `:claims` means the service will look up a `Claims::User` by default.